app/views/provider_interface/organisations/show.html.erb

```diff
@@ -7,7 +7,13 @@
       </h2>
       <div class="app-banner govuk-!-margin-bottom-4">
         <div class="app-banner__message">
-          Contact <%= permissions.training_provider.name %> to change permissions.
```

Comment:
I probably misunderstood the card but I would think that we need a conditional here to check if permission has been set up.

```
if permissions.setup_at.nil?
  "<%= permissions.training_provider.name %> have not set up permissions yet..."
else
  "Contact <%= permissions.training_provider.name %> to change permissions."
 end
```

Reply:
I think this is what I was unsure about and mentioned earlier, and I think you're right, there are 2 cases, one for when permissions have yet to be set and one when they've been set, thanks - will amend :+1:

Reply:
https://github.com/DFE-Digital/apply-for-teacher-training/pull/2730/commits/5ee272d15c3f8f7ef6f4bfb5c23e29afbb274291 conditionally shows the different messages, it seems awkward that they are not in the same component though, will investigate bringing them into the same place.